app/js/components/identifiers/addIdentifier.jsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import PropTypes from 'react-proptypes';
+import apiCall from '../../utilities/apiHelper';
+
+class NewIdentifier extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            newState: false,
+            addIdentifiers: {},
+            editErrors: {
+                uuid: '',
+                error: '',
+            },
+            addErrors: {
+                uuid: '',
+                error: '',
+            },
+        };
+        this.callNew = this.callNew.bind(this);
+        this.callCreate = this.callCreate.bind(this);
+        this.handleCreateChange = this.handleCreateChange.bind(this);
+        this.callCancelPromise = this.callCancelPromise.bind(this);
+    }
+
+    callNew() {
```

Comment:
Can you use a more descriptive name for this method? Seeing that the method has no comment tied to it, a descriptive name would aid other understand its use